package.json

```diff
@@ -6,7 +6,7 @@
    "scripts": {
       "test": "mocha -r ts-node/register test/**/*.test.ts",
       "tsc": "tsc",
-      "start": "ts-node build/app/app.js"
+      "start": "ts-node build/app/startapp.js"
```

Comment:
It is common practice to always camelCase files and PascalCase files with classes :)